[2024-12-31T10:44:56.620+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-31T10:44:56.664+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: final_backup1.initialize_data manual__2024-12-31T10:32:00.554374+00:00 [queued]>
[2024-12-31T10:44:56.677+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: final_backup1.initialize_data manual__2024-12-31T10:32:00.554374+00:00 [queued]>
[2024-12-31T10:44:56.678+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-12-31T10:44:56.700+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): initialize_data> on 2024-12-31 10:32:00.554374+00:00
[2024-12-31T10:44:56.711+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=210) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-31T10:44:56.711+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'final_backup1', 'initialize_data', 'manual__2024-12-31T10:32:00.554374+00:00', '--job-id', '1046', '--raw', '--subdir', 'DAGS_FOLDER/final_backup1.py', '--cfg-path', '/tmp/tmpzk90h9z_']
[2024-12-31T10:44:56.713+0000] {standard_task_runner.py:72} INFO - Started process 228 to run task
[2024-12-31T10:44:56.714+0000] {standard_task_runner.py:105} INFO - Job 1046: Subtask initialize_data
[2024-12-31T10:44:56.739+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-31T10:44:56.799+0000] {task_command.py:467} INFO - Running <TaskInstance: final_backup1.initialize_data manual__2024-12-31T10:32:00.554374+00:00 [running]> on host 05c0118fb254
[2024-12-31T10:44:56.948+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='final_backup1' AIRFLOW_CTX_TASK_ID='initialize_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-31T10:32:00.554374+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-31T10:32:00.554374+00:00'
[2024-12-31T10:44:56.951+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-31T10:44:56.957+0000] {final_backup1.py:117} INFO - Initializing workflow state...
[2024-12-31T10:44:56.958+0000] {final_backup1.py:119} INFO - Workflow initialized with state: {'workflow_state': 'initialized', 'files_to_copy': {}}
[2024-12-31T10:44:56.959+0000] {python.py:240} INFO - Done. Returned value was: {'workflow_state': 'initialized', 'files_to_copy': {}}
[2024-12-31T10:44:56.995+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-31T10:44:56.996+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=final_backup1, task_id=initialize_data, run_id=manual__2024-12-31T10:32:00.554374+00:00, execution_date=20241231T103200, start_date=20241231T104456, end_date=20241231T104456
[2024-12-31T10:44:57.051+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-31T10:44:57.121+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-31T10:44:57.124+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
